production/helm/loki/templates/backend/service-backend-headless.yaml

```diff
@@ -8,7 +8,13 @@ metadata:
   namespace: {{ $.Release.Namespace }}
   labels:
     {{- include "loki.backendSelectorLabels" . | nindent 4 }}
+    {{- toYaml .Values.loki.serviceLabels | nindent 4}}
+    {{- toYaml .Values.singleBinary.service.labels | nindent 4 }}
```

Comment:
why are we putting the single binary's service labels on the backend service? should this be backend instead?

Reply:
thx, fixing